packages/story-editor/src/components/panels/document/taxonomies/FlatTermSelector.js

```diff
@@ -110,6 +125,19 @@ function FlatTermSelector({ taxonomy, canCreateTerms }) {
     [taxonomy, termCache]
   );
 
+  useEffect(() => {
+    (async function () {
+      const results = await addSearchResultsToCache(taxonomy, {
```

Comment:
just for my own edumacation: what's the benefit of using an IIFE vs not?

Reply:
just to do async operations in the scope of an effect with async/await instead of traditional promises. I'm only doing it cus I thought it was preferred to use async/await vs promises stylistically amongst the teams.